samples/1.0.0-rc1/HelloWeb/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/aspnet:1.0.0-rc1
+
+COPY project.json /app/
```

Comment:
Change this to just copy the app.

COPY . /app./

and then remove the other copy line. Since beta8 shipped doing this two-stage copy can introduce subtle bugs.